src/EditorFeatures/CSharpTest/EditAndContinue/StatementEditingTests.cs

```diff
@@ -6772,6 +6772,34 @@ public void LocalFunction_ReadOnlyRef_ReturnType_Update()
                 Diagnostic(RudeEditKind.ReadOnlyReferences, "local", FeaturesResources.local_function));
         }
 
+        [Fact]
+        public void LocalFunction_ChangeToStatic_Update()
```

Comment:
>public void LocalFunction_ChangeToStatic_Update() [](start = 8, length = 49)

Also add test that adds/removes `unsafe` modifier. That should be allowed too. 
Another modifier that needs testing is `async` - we should report a rude edit, which we don't. [https://github.com/dotnet/roslyn/issues/37054](https://github.com/dotnet/roslyn/issues/37054)
Please add a test that's skipped with a ling to the issue.